### PR TITLE
chore(repo): Add postpublish purge-cache script, eslint env addition,…

### PR DESCRIPTION
**Pull request by igneel64**
## Type of change

- [ ] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [x] other:

## Packages affected

- [ ] `@clerk/clerk-js`
- [ ] `@clerk/clerk-react`
- [ ] `@clerk/nextjs`
- [ ] `@clerk/types`
- [ ] `@clerk/clerk-expo`
- [ ] `@clerk/backend-core`
- [ ] `@clerk/clerk-sdk-node`
- [ ] `@clerk/edge`
- [x] `build/tooling/chore`

## Description

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.

A few publish related chores:
1. @clerk/backend-core description field
2. Eslint env field addition
3. postpublish cache purge for @clerk/clerk-js
4. Correct package manager specification for [npm fix](https://github.com/npm/cli/pull/4371). 
5. Fix updating the package-lock versions before commiting new versions.

### CODE REVIEW

File: packages/backend-core/package.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "name": "@clerk/backend-core",
   "version": "0.4.1",
   "license": "MIT",
+  "description": "Core Clerk API resources and authentication utilities for JavaScript environments.",
```
Comment (SokratisVidros) [inline packages/backend-core/package.json:5]: 🙃 
```suggestion
  "description": "Clerk Backend API core resources and authentication utilities for JavaScript environments.",
```

File: packages/types/package.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "@clerk/types",
   "version": "1.25.2-staging.0",
   "license": "MIT",
-  "description": "Clerk.dev Types",
+  "description": "Typings for Clerk.dev libraries.",
```
Comment (SokratisVidros) [inline packages/types/package.json:5]: 🙃 

```suggestion
  "description": "Typings for Clerk libraries.",
```

**Review by SokratisVidros — APPROVED** [review]
🎖️

**Review by nikosdouvlis — APPROVED** [review]
💯

**Pull request merged**